torch/distributed/fsdp/_optim_utils.py

```diff
@@ -19,10 +19,11 @@
 import torch.distributed as dist
 
 # Import the entire FSDP file to avoid circular imports
-import torch.distributed.fsdp.fully_sharded_data_parallel as FSDP
+import torch.distributed.fsdp.fully_sharded_data_parallel as fsdp_file
```

Comment:
Rename `FSDP` to `fsdp_file` since sometimes we import `FullyShardedDataParallel` as `FSDP`.